.github/workflows/tests.yml

```
@@ -76,6 +80,12 @@ jobs:
           sudo apt install -y -q ${{ matrix.mysql-client }}
           mysqldump --version
 
+      - name: Install mongodb database tools
+        run: |
+          wget https://fastdl.mongodb.org/tools/db/mongodb-database-tools-debian92-x86_64-100.3.1.deb
+          apt install ./mongodb-database-tools-*.deb
```

```suggestion
          sudo apt install ./mongodb-database-tools-*.deb
```

It should work. :crossed_fingers: